src/client/components/Footer/Footer.css

```diff
@@ -0,0 +1,109 @@
+body {
+    margin:0;
+    padding:0;
+    height:100%;       
+   }  
+
+.footer {  
+    position: absolute;        
+    height: 105px;
+    width: 100%;
+    left:0px;   
+    background-color:#ffffff;
+    box-shadow: 0px 0px 20px rgba(0, 0, 0, 0.5);  
+    bottom: 0;
+    min-width:1100px;    
+  }
+
+  .footer p {
+    position:absolute;
+    height: 21px;
+    width: 162px;
+    left: 139px;
+    border-radius: nullpx;
+    background-color:rgb(255, 255, 255);
+    font-family: Roboto;
+    font-size: 18px;  
+    font-style: normal;
+    font-weight: 400;
+    line-height: 21.09px;
+    letter-spacing: 0em;
+    text-align: left;
+    color:black;
+    margin-top:50px;
+}
+
+.footer ul {
+    position: absolute;
+    left: 1030px;
+    margin-top:50px;
+    margin-left:20px;
+    margin-bottom: 40px;
```

Comment:
You have **lots** of fixed pixel values. I would encourage you to remove most of them. Im quite sure this layout will break on mobile. Like left `1030px` is a dangerous layouting